includes/addons.php

```diff
@@ -71,6 +71,45 @@ function pmpro_getAddons() {
 	return $addons;
 }
 
+/**
+ * Search installed addons with wrong folder name.
+ *
+ * @since TBD
+ */
+function pmpro_searchWrongFolderNameAddons() {
```

Comment:
Is this the best name for this function.

We've been inconsistent on this in the past, but basically any new function we had should use snake case, e.g. pmpro_search_wrong_folder_name_addons().

But then I also think pmpro_find_wrong_folder_name_addons() might be more descriptive? Is there something even better? What other contexts are we going to use this? We only use this once, so we're abstracting a little earlier, but I think it's okay here. Let's just imagine how it might be used going forward and have the function name and doc block make the most sense.

Thanks!